docs/user-manual/federation-address.adoc

```diff
@@ -83,19 +82,19 @@ Sample Address Federation setup:
         <upstream name="eu-east-1">
            <static-connectors>
               <connector-ref>eu-east-connector1</connector-ref>
-              <connector-ref>eu-east-connector1</connector-ref>
+              <connector-ref>eu-east-connector2</connector-ref>
```

Comment:
You have changed connector refs in almost every config example in the doc, fixing some apparent bugs within themselves...however those config examples all seem to remain inconsistent with each other in terms of refs. Varying connector name formats like "connector1", "eu-east-connector2", and "eu-east-1-connector" all remain. Should those that seem related to an overall geographic example perhaps actually be aligned to use the same naming format?

Reply:
TBH I don't know. I consider xml excerpts as isolated examples. 
If I am wrong, probably original author/s should fix appropriately?